js/index.js

```diff
@@ -1,6 +1,100 @@
+
+function countWords(str) {
+    let totalWords = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str[i] === ' ') {
+            totalWords++
+        }
+    }
+    return totalWords;
+}
+
+console.log(countWords(text));
+
+// counting the characters 'et'
+function countEt(str) {
+    let totalWords = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str[i] === 'e' && str[i+1] === 't') {
+            totalWords++
+        }
+    }
+    return totalWords;
+}
+
+console.log(countEt(text));
+
+// Bonus 2
+let phraseToCheck = '';
+
+function findPalindrome(str) {
+
+    let strLowerCase = str.toLowerCase;
+    let strLenght = str.length;
+    
+    for (let i = 0; i < strLenght / 2; i++)  {
+        if (strLowerCase[i] === strLowerCase[strLenght - 1 - i]) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+}
+
+console.log(findPalindrome("step on no pets"))
```

Comment:
Last bonus isn’t working, try and take a look at the recommended documentation on the lab instructions.